libs/utils/analytics.js

```diff
@@ -0,0 +1,52 @@
+// Analytics
+// Include Adobe analytics tracking attributes to the DOM
+// ---
+// 1) Link Hierarchy Attribute - (daa-lh) (daa-im) decorateBlockAnalytics()
+// daa-im
+// - Simple attribute flag if set
+//   to true will capture impressions
+//   for each link in that container.
+// daa-lh
+// - Hierarchy values that are used to
+//   create a complete view of where each
+//   interaction is located. Our code will
+//   combine hierarchy values to create
+//   where(unique identifier) for every interaction.
+//
+// 2) CTA Attribute - (daa-ll) decorateLinkAnalytics()
+// (CTA English value-CTA number in current box)
+// daa-ll
+// - Interaction identifier. Such as
+//   "Learn more" for a learn more link.
+// ---
+// PRD: Home Page Link Tracking...
+// [https://wiki.corp.adobe.com/display/~cwest/PRD%3A+Home+Page+Link+Tracking]
+
+// expects block el
+export function decorateBlockAnalytics(el) {
```

Comment:
nit: Instead of comment, use more descriptive name:

```suggestion
export function decorateBlockAnalytics(blockEl) {
```